modules/ROOT/pages/endpoints.adoc

```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-=======
+// ====
+
+== Delete Endpoints
+
+From either the Partner or Host page, you can delete any endpoints that are not used in message flow configurations.
```

Comment:
GLOBALLY end a fragment or sentence that leads directly to a graphic, example, list, or step with a colon.
```suggestion
From either the Partner or Host page, you can delete any endpoints that are not used in message flow configurations:
```

Reply:
Fixed.